dbfit-java/informix/TestDbConnectionDbFitInformixINFORMIX.properties

```diff
@@ -0,0 +1,4 @@
+service=localhost:16466
+username=dbfit
+password=DBFIT
+database=dbfit_ifx
```

Comment:
I think we should remove these `.properties` from version control too and keep `.default` scripts only? `.gitignore` should be updated (similarly to Oracle props entries which we have now).